docs/install/create-project.md

```diff
@@ -0,0 +1,457 @@
+---
+myst:
+  html_meta:
+    "description": "Create a Plone project"
+    "property=og:description": "Create a Plone project"
+    "property=og:title": "Create a Plone project"
+    "keywords": "Plone, Plone 6, create, project, install, cookiecutter"
+---
+
+
+(create-a-project-label)=
+
+# Create a project
+ 
+This chapter describes how you can create a web application project using Plone, with full control over development and deployment.
```

Comment:
Would it make sense to add something along these lines to redirect people are actually intending to make a contribution?

"This is the correct path to follow if you want to build a website using Plone. If you'd like to make changes to Plone itself, follow the contributor documentation [link] instead."

Reply:
Yes, of course, a cross-reference, especially for those who land here first, is needed. Fixed in 070ea528e971db53ff56d08b1ce7e4a5406bd5ff